lyricsheets/fonts/fonts.py

```diff
@@ -32,19 +42,45 @@ def split_by_rendered_width(
 
         return ans
 
+    def get_length(self, text: str) -> int:
+        width = 0
+        if self.style.spacing:
+            for c in text:
+                extent = self.dc.GetTextExtent(c)
+                scaling = (
+                    self.style.fontSize
+                    * PRECISION_SCALE
+                    / (extent.height if extent.height > 0 else 1)
+                )
+                width += (extent.width + self.style.spacing) * scaling
+        else:
+            extent = self.dc.GetTextExtent(text)
+            scaling = (
+                self.style.fontSize
+                * PRECISION_SCALE
+                / (extent.height if extent.height > 0 else 1)
+            )
+            width = extent.width * scaling
+
+        return int(round(self.style.scaleX / 100 * width / PRECISION_SCALE, 0))
+
 
-class FontNotFoundException(Exception):
-    pass
+def _find_font(style: Style) -> wx.Font:
+    return _find_font_cached(
+        style.fontName, style.fontSize, style.isBold, style.isItalic, style.isUnderline
+    )
 
 
 @cache
-def _find_font(fontName: str, fontSize: int) -> ImageFont.FreeTypeFont:
-    for font in findSystemFonts():
-        try:
-            f = ImageFont.FreeTypeFont(font)
-            if f.getname()[0] == fontName:
-                return ImageFont.truetype(font, fontSize)
-        except:
-            continue
-
-    raise FontNotFoundException(fontName)
+def _find_font_cached(
+    fontName: str, fontSize: int, isBold: bool, isItalic: bool, isUnderline: bool
+) -> wx.Font:
```

Comment:
Missing strikethrough (it's also missing in the Aegisub wx code but is present in the win32 code)

Reply:
FIxed